### PR TITLE
Lobbies UI UX updates

### PR DESCRIPTION
**Pull request by AngieDutra**
This PR:

- reverts lobbies list arregement
- makes the entire lobby container clickable
- Deletes scroll bar without deleting the scroll function
- Adds a "no lobbies yet" text

### CODE REVIEW

**Review by tvillegas98 — DISMISSED** [review]
Nice improvements 😄 .

File: client/Assets/Scripts/UIManager.cs [hunk 1]
```diff
@@ -20,16 +23,19 @@ void Update()
...
+            noLobbiesText.SetActive(false);
             GenerateList(LobbyConnection.Instance.lobbiesList, lobbyItemPrefab, lobbiesContainer);
             lobbiesEmpty = false;
         }
+        noLobbiesText.SetActive(lobbiesEmpty);
     }
 
     public void GenerateList(List<string> itemList, Object itemPrefab, Transform container)
     {
         itemList.ForEach(el =>
```
Comment (tvillegas98) [inline client/Assets/Scripts/UIManager.cs:35]: I think before entering the foreach, you cando `itemList.Reverse();`

Comment (tvillegas98) [reply client/Assets/Scripts/UIManager.cs:35]: And avoid having changing the list arrangement in each iteration

Comment (AngieDutra) [reply client/Assets/Scripts/UIManager.cs:35]: will do!

**Review by BertovDev — CHANGES_REQUESTED** [review]

**Review by ncontinanza — APPROVED** [review]
LGTM! 🚀

**Review by BertovDev — APPROVED** [review]
Looking and Working good! 🤩

**Pull request merged**